src/main/java/cow/commands/FindCommand.java

```diff
@@ -0,0 +1,37 @@
+package cow.commands;
+
+import cow.exceptions.CowExceptions;
+import cow.filesaver.FileSaver;
+import cow.message.Ui;
+import cow.todolist.TodoList;
+
+// solution below inspired by https://github.com/se-edu/addressbook-level2/tree/master
+
+/** Creates a Find command object. **/
+public class FindCommand extends Command {
+    public static final String COMMAND_WORD = "find";
+    public static final String COMMAND_EXAMPLE = "todo book";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Finds all task with argument in the description.\n"
+            + "Example: " + COMMAND_EXAMPLE;
```

Comment:
Great that you adhered to the coding standard of having line breaks before an operator